tensorflow/compiler/xla/service/llvm_ir/kernel_support_library.cc

```diff
@@ -98,8 +98,10 @@ void KernelSupportLibrary::EmitAndCallOutlinedKernel(
     ir_builder->SetInsertPoint(return_inst);
 
     std::vector<llvm::Value*> arg_values;
-    std::transform(function->arg_begin(), function->arg_end(),
-                   std::back_inserter(arg_values), std::addressof<llvm::Value>);
+    for(llvm::Function::arg_iterator arg = function->arg_begin(),
```

Comment:
This can be a range for as `for (llvm::Value& arg : function->args())`.

There also needs to be a space after `for` as per the google style guide, but just running clang-format should fix that for you.

Reply:
I had some issues with the range for loop so had to implement it this way for best compatibility. I will update the PR after running through clang-format.

Reply:
Ok, can you please add a (brief) comment with the error you get on OSX with 

 - `std::transform`
 - Range for

Otherwise I think it is only a matter of time before someone "cleans this up" to use one of the above.

Reply:
makes sense. I added a comment with a link to the issue. The issue has the details. LMK what you think